content/influxdb/v2.0/upgrade/v1-to-v2/_index.md

```diff
@@ -136,6 +136,9 @@ Consider [enabling authentication in your InfluxDB 1.x instance](/influxdb/v1.8/
 **before** upgrading to InfluxDB 2.0 to ensure the appropriate credentials are in place and
 that the various applications, agents, and visualization tools are able to connect.
 
+If `auth-enabled = false` is set in your InfluxDB 1.x configuration, after upgrading to InfluxDB 2.0,
```

Comment:
If they set `auth-enabled = true`...does this resolve credentials issue, or are there additional steps to ensure their credentials will work after upgrading?